packages/ui-bindings/Agent/agentInterface.tsx

```diff
@@ -47,14 +51,14 @@ export interface Agent {
   type: string
   image: string
   note: string
-  ownedEconomicResources: [number]
-  agentProcesses: [number]
-  agentPlans: [number]
-  agentEconomicEvents: [number]
-  agentCommitments: [number]
-  agentRelationships: [number]
-  agentRoles: [number]
-  agentRecipies: [number]
+  ownedEconomicResources: [EconomicResource]
+  agentProcesses: [Process]
+  agentPlans: [number] // TODO replace with [Plan]
+  agentEconomicEvents: [number] // TODO replace with [EconomicEvent]
+  agentCommitments: [number] // TODO replace with [Commitment]
+  agentRelationships: [AgentRelationship]
+  agentRoles: [AgentRelationshipRole]
+  agentRecipies: [number] // TODO replace with [ResourceClassification]
```

Comment:
We should make it a point as we're adding these missing Objects to go back and remove any existing TODOs (just an observation)

Reply:
Also, is there a reason why these are number types instead of any?

Reply:
I am treating them as an array of Id's, but they should be any. thanks